etl/scripts-ccao-data-raw-us-east-1/spatial/spatial-kriging_surfaces.R

```diff
@@ -634,38 +672,44 @@ create_av_kriging <- function(data, raster_file){
   clean_data <- rbind(clean_data, aps)
   clean_data <- unique(clean_data)
 
-  v = variogram(noise~ 1, clean_data, cutoff = 200000, width = 500)
-  v.m = fit.variogram(v, vgm(1, "Gau", 5000, 1))
-  k = krige(noise ~ 1, clean_data, raster_file, v.m)
+  v <- variogram(noise ~ 1, clean_data, cutoff = 200000, width = 500)
+  v.m <- fit.variogram(v, vgm(1, "Gau", 5000, 1))
+  k <- krige(noise ~ 1, clean_data, raster_file, v.m)
 
   return(k)
 }
 
 
-#Year-by-year with best performing model:
-years <- c("2010", "2011", "2012", "2013", "2014", "2015", "2016", "2017",
-           "2018", "2019", "2020")
+# Year-by-year with best performing model:
+years <- c(
+  "2010", "2011", "2012", "2013", "2014", "2015", "2016", "2017",
+  "2018", "2019", "2020"
+)
 
-out <- lapply(years, create_year_kriging, raster_file = rasters_100, data = airport)
+out <- lapply(
+  years, create_year_kriging,
+  raster_file = rasters_100, data = airport
+)
 
-#Demo from 2010:
-#plot_surface(out[[1]])
+# Demo from 2010:
+# plot_surface(out[[1]]) # nolint
```

Comment:
**[Nitpick, non-blocking]** Same here:

```suggestion
```

Reply:
I think you're right that this should get removed, but this entire file needs to be cleaned up for extraneous code. I *only* wanted to address formatting in this PR so that there wouldn't be any uncertainty as to whether the changes here might lead to any changes in outputs or how code is understood.

Reply:
@jeancochrane https://github.com/ccao-data/data-architecture/issues/679